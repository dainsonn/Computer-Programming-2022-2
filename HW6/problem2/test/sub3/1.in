root
pivot@eee
recommend 3
exit
