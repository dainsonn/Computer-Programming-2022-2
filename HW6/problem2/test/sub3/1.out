------ Authentication ------
id : passwd : ----------------------------
root@sns.com
post : Post contents
recommend <number> : Recommend <number> interesting posts
search <keyword1> ... <keywordN> <threshold> : List post entries whose contents contain keywords more than the threshold
exit : Terminate this program
----------------------------
Command : ----------------------------
id : 163
created at : 2019/02/20 00:00:00
title : then caviar alert are is 
content :
buteo
then am are cornish is he
he my my am a the adroit are he are you is you he dreamless
where you chevalier then is where her my then the you you creation the cynocephalidae are why biscuits 
alexandria auriga befoul derived cellulosid duet concious dachau disinterestedness essaying
bh anxiousseat acrocarp dividers coleonyx advance(a) dolphin constipate agog deprivation
carambola displeasure exploit attuned assistive baccharis barnacled aegyptopithecus amplification boogie
bougie eau cresol cajun curmudgeon acrocephalus cocktail catechism bugle carvel-built
coiffure anergy endodontics effluxion expectant bassariscidae curassow alter banister cinclus
apodeme blackhead cocoa chaetal contextual araba amaryllidaceae deceleration attribution breathe
beaucoup ablepsia coverture beryl cucurbita betula associative accepting catechistic euproctis
adelomorphous balldress brand-newness catacomb basidiolichen brunt carpet attainder crape centerpiece
deluge arboraceous agouti chaffe dualistic adherent bolt-hole bannerlike absento contra
badli anguidae embalmer effort anneal drinkable annunciation celestite entice coverall
asio bullfighter boiler chirrup dreggy banzaijapanese appointments eutamias contuse burgle
amaryllidaceae exceptional canape communist cunningman carver abdication bodybuilding chigoe amid
diapensiaceae aneurysm carnosaur crackajack campaign enough affluxion competition beg crutched
coffeeberry cogitate blinds crossroads disfigurement abscision ditch diluent devote androgenesis
dilemma danae clathrus displeasure dissembler contortion bushes coup counterscarp bravo
cancellation eventual droit broker denigration cowering(a) ardently diclinous chaenactis anglophile
baneful bipinnatifid corridors coffer altarstairs dingdong downfield decide earthstar detestably
downstream boom entry essaying extinguisher behind aristarchy bowfin bulbaceous cusped
beams boater expectantly econometrics disenable backyard blephilia breaks byway conjecturality
discretional arithmetic booted awkwardness burgh drippy dishevel conclude altarstairs annealing
craichy excogitation discompose dimethylglyoxime cocker enterotoxemia alterable eggnog disinterest corncrake
alnaschar blenheim conjecturality benzol discompose badv ascending(a) apodeme destructibility croft
blowback discontentment devitedness bighorn columbararium corncrake disdainful actuator catchy chaetal
blackleg cornish clinically bride badgering disdainful darlingtonia aller displeasure emaciated
bartlett bavin apace chirrup explosion escapement beaucoup diacoustics cerni ceaselessness
dhobi dialogism achondroplasia dipteronia allylic conover achomawi dextrad axially aphrodite
dyslexia clog earthquake bibliography afro all(a) cuniculus bumpkinly dogmatize cairngorm
basidiomycete aidless conover comprehensively camarista dome eveille assassination excellence bout
disconcerting bawcock allegedly is einsteinian dissonance ejectment bumbling alleviate carnal
acanthosis convey confabulation cro-magnon actinomyxidian escopet dissembling diplomat anaphoric brazen
consolidate blustering(a) erectness disembark elymus chromeyellow excel cephalotaceae ciconiiformes enologist
expire bobbish espousals cudweed
----------------------------
id : 167
created at : 2018/12/19 00:00:00
title : erosion clown dixit bloodwort a 
content :
the corpse experimentist dissolving then is her convector are her you corrected ended befog her is conceit ethology where cellulose her is are why you alfardaws are is cowering(a) am a central-fire
why desperandum he
why you am her her 
botulinal craichy branchiostegidae dissipate certify bravo anticlimactic drumhead dracula adding
antigua dragon barcarole aerobics bespeckle bioluminescent dewey briquet commination comprendre
burmese condemnation epiphyllum burrito blustering curbstone corn-fed apportioning deprecatory epauliere
abest eram bread chorography exclusive basidial cabinetmaking blucher extinct arbitral
duodenary chaetognatha dialectical asclepiadaceous credula exasperatingly dizygotic borrelia continued cabinet
badv asynergy deposition archebiosis avowed(a) aground calostomataceae attuned creativity chapel
brooklet adverbially cavity barbiturate ergotropism erg contemporaneously avoirdupois cheapskate borax
dry-cleaned catostomid ebb dour diverting claymore cookshop destructive-metabolic(a) bookdealer bullocky
excluding explicit etching cortaderia chlorophoneus accension bignoniaceae aeronaut bedevilment euascomycetes
demonism desmodontidae bentley albania cooky constantan anglicanism applecart canst clean-cut
amoralism christening compartment brakeman bimillenial cabinet cetacea copartner descending enunciation
dominant beaming cachet departing admonisher bounteousness cross-modal devastate cons aftercome
crucial cotula agrologic cotswolds antacid conglobation adorn beanstalk distingue arbitration
contemporaneously diphylla comeliness bottomless cornflower antidorcas crusade asymptotically berliner bacteriochlorophyll
blustering blistered bakers ascending(a) bipinnatifid aground antipollution crith approachable agoraphobic
amor afterpart anachronic canal apodal equidistant dextrad cerium dispossess aggravable
ambitiously downhill complaining(a) communist exonerate chartaceous davus alarmism definable coelum
cerecloth dilemma dialectics enterotoxemia cynic adductive divestiture doodad branchiobdellidae conventionalism
bimonthly branchiobdellidae arguments bluish becloud egyptology ambivalence everglades dishwater anapest
bioluminescent arrested akkadian amari aboveboard back(a) brian conversationalist degenerate disburden
denticulate boundaries christening beslubber birdwitted corduroy aversion blackguard bonanza devise
burglarious babylonia discontentment buffeted elaphe boliviano abvolt cereals betraying complaining(a)
disenable addiction chloroformed accusable bap anisogametic bromelia alluvial bolbitis anabolism
certify bailiwick controversialist arcadia epimedium aggravable advanced cross-reference credula dissepiment
coelenteron campagne deuteranopic anticlimactic confectionery eldritch angiology ecballium bankruptcy acromegaly
ableness acumination abhorrence biliary canonist capricious countermark condemnation commensurateness breastwork
blanched deparia chlorosis decuration congregate asclepias exempie boomer billiards acting(a)
esophageal deep-mined audible antibacterial eroded exert abfarad degradation blurt degradation
cirrostratus epicurism albatrellus erroneous ecclesiastical campylorhynchus acrocarpus achromatize dumfound bradypodidae
ascending disburden alamo correctly despite allegedly atoms carotenemia arrive affair
bap bibliopolic ablepsia cairngorm descendent agrostemma briquet chimariko caryatid clickety-clack
berth decoration charcoal ephesian efforts equidistant coiling adeology
----------------------------
id : 72
created at : 2018/08/03 00:00:00
title : are bidder concertina est co 
content :
beechnut he he then correspondingly then the why my the appurtenance cosmic am he
am you am my
the am ardeidae you aduncity the are enthusiastic am
elaeocarpus you my why aphonia her you dozen a her are ell disbursement 
aix diddle doxology dextrad cogitable adamant contemporaneously evlugate certifiable distraught
demeanor clothesless ascosporic chawbacon cupping disburden diversionary disenthrallment contrariwise compote
cenobite excogitate amniota aslant disobedience dhu cuniculus eruption conscia erysiphe
barratry applicable disdainfully aisle bondsman cpoetry centromere avert checkbook couvert
durmast acquiring broodmare blandishment clothed bonus congenial cauterization disembark chemisorptive
antichristian colori casing carnivorous eripuit cufflink chinless emo breadline disconcerting
aegyptopithecus cairngorm conflagration electorate actinomyxidia executors e'en barrenness enrollment benchmark
colloquy astound embellishment andean bourgeon brazilian diverge assimilable double-crosser depth
cerecloth brachial abatjour custodiet demonstrate aluminum exegetic advertiser chub euplectella
exempla capnomancy corsage cactaceae entice alternateness dislike descriptively daring cresistance
acephalia atilt corsage braggart drippy centrality capacitive arthrospore allabsorbing achondroplasia
cellulosid explicit birdie crosscheck berkelium doggo bladder chirurgical audacity economizer
environmentalist cathedra ascending(a) albuterol bathos denunciatory daring ephippiorhynchus darlingtonia apterygiformes
aggressive aquamarine epic costochondritis barratry anaphora cultivated aerostatic crith armadillo
bh arrive aquamarine
----------------------------
root@sns.com
post : Post contents
recommend <number> : Recommend <number> interesting posts
search <keyword1> ... <keywordN> <threshold> : List post entries whose contents contain keywords more than the threshold
exit : Terminate this program
----------------------------
Command : 